Makefile

```diff
@@ -40,7 +40,7 @@ build: go.sum
 
 .PHONY: build-binaries
 build-binaries: guard-SEMVER
-	./scripts/build-binaries.sh ${SEMVER} '$(BUILD_TAGS)' '$(ldflags)'
+	./scripts/build-binaries.sh ${SEMVER} '$(BUILD_TAGS)' '${ldflags}'
```

Comment:
Why is BUILD_TAGS using `()` but this not? We're using `$(ldflags)` in another spot above.